src/clj_http/client.clj

```diff
@@ -141,7 +141,7 @@
   "Resolve and apply cheshire's json stream decoding dynamically."
   [& args]
   {:pre [json-enabled?]}
-  (apply (ns-resolve (symbol "cheshire.core") (symbol "decode-stream")) args))
+  (doall (apply (ns-resolve (symbol "cheshire.core") (symbol "decode-stream")) args)))
```

Comment:
Is there a reason why you chose not to switch this to use `parse-stream-strict` also?

Reply:
I was not sure if there if the `parse-stream-strict` has other differences in functionality besides eagerness/lazyness.

Reply:
Looking at the doc string of ‘parse-stream-strict’ it looks like it’s designed for this use-case.

Reply:
@dakrone I just realized that `parse-stream-strict` is not part of cheshire 5.8.1. Can we get a new release of `cheshire` so it can be used in clj-http?